addons/test_mail/tests/test_mail_followers.py

```diff
@@ -991,8 +996,11 @@ def setUpClass(cls):
     def _post_message_and_get_unfollow_urls(self, record, partner_ids):
         """ Post a message on the record for the partners and extract the unfollow URLs. """
         with self.mock_mail_gateway():
-            record.message_post(body='test message', subtype_id=self.env.ref('mail.mt_comment').id,
-                                partner_ids=partner_ids.ids)
+            record.message_post(
```

Comment:
Here, the tests expects the mails to have an unsubscribe link. I hence adapted the tests to make sure that the emails includes the footer containing that link.